packages/vuetify/src/components/VMenu/VMenu.ts

```diff
@@ -432,7 +452,7 @@ export default baseMixins.extend({
         this.$nextTick(() => activator && activator.focus())
       } else if (
         !this.isActive &&
-        [keyCodes.up, keyCodes.down].includes(e.keyCode)
+        [keyCodes.up, keyCodes.down, keyCodes.home, keyCodes.end].includes(e.keyCode)
```

Comment:
https://www.w3.org/TR/wai-aria-practices/#menubutton only mentions up/down, and none of the sections you linked explicitly state that home/end should also open the menu.

Reply:
You are absolutely correct! While focusing a select, home and end should only select the first/last selectable item and should not open the menu.
Sorry for the inconvenience. I will fix this as soon as I can.